core/api/src/app/wallets/get-invoices-for-wallets.ts

```diff
@@ -25,3 +26,34 @@ export const getInvoicesForWallets = async ({
     paginationArgs,
   })
 }
+
+export const getInvoicesForWalletsByExternalIdSubstring = async ({
```

Comment:
Is this a fuzzy search for a part of an external id? Is this efficient on DB level?

Reply:
It's to potentially support a user who wants to segregate use-cases by id for a single wallet. E.g. `service1-<uuid>`, `service2-<uuid>` etc.

It won't be as efficient as an exact match, but from testing on signet db it takes around 1s.

Now that I think about it though, we probably should explicitly signal the param for a fuzzy search on the API and not overload the exact search  if we decide to keep this in.

Reply:
I opted to remove these methods since they're not straightforward to add to the API in a clean way and the exact need isn't known as yet. We can potentially revisit later https://github.com/GaloyMoney/galoy/pull/4262/commits/34dfa15a9964048e8bb4b7ddd80cc11afe5d770f